Protocol.md

```diff
@@ -126,10 +126,10 @@ ReactiveSocket frames begin with a ReactiveSocket Frame Header. The general layo
     +-------------------------------+
 ```
 
-* __Stream ID__: (32) Positive signed integer representing the stream Identifier for this frame or 0 to indicate the entire connection.
+* __Stream ID__: (32 bits = max value 2^31-1 = 2,147,483,647) Signed positive Integer representing the stream Identifier for this frame or 0 to indicate the entire connection.
```

Comment:
I put that in specifically for 32 and 64 bit because they hit the max sizes and want to be explicit.